.npmignore

```diff
@@ -1,16 +1,24 @@
 # standard ignore
+*config*
+*conf*
+
+.github/
 .gitignore
+.eslintrc
 .stylelintrc
-karma.conf.js
-rollup.config.js
-babel.config.js
-.github/
-index.html
+
+coverage/
+docs/
 scripts/
 test/
-demo/css
-demo/sass
-coverage/
+demo/**/*.css
+demo/**/*.scss
+demo/**/*.html
+
+dist/es5.js
+
+index.html
 
-# ignore src
-src/**/*
+# ignore src css & scss
+src/**/*.scss
+src/**/*.css
```

Comment:
Be sure to REBASE off of the latest release of auro-icon and ensure that this code remains true to your commit here.